wp-content/themes/pub/wporg-learn-2024/src/style/_sensei.scss

```
@@ -136,6 +136,61 @@ body.sensei {
 			border: solid 1px var(--sensei-secondary-color);
 		}
 	}
+
+	section.wp-block-sensei-lms-course-results {
+		display: flex;
+		flex-direction: column;
+		margin-top: 40px;
+		margin-bottom: 40px;
+
+		> h2 {
+			text-align: left;
+			font-size: var(--wp--preset--font-size--heading-6);
+			margin-top: 0;
+			margin-bottom: var(--wp--preset--spacing--20);
+		}
+
+		> section.wp-block-sensei-lms-course-results__module {
+			margin-bottom: var(--wp--preset--spacing--20);
+			border-radius: var(--wp--custom--button--border--radius);
+
+			header {
+				padding-top: var(--wp--preset--spacing--20);
+				padding-bottom: var(--wp--preset--spacing--20);
+
+				h3 {
+					margin: 0;
+				}
+			}
+		}
+
+		> ul.wp-block-sensei-lms-course-results__lessons {
+			margin: 0;
+
+			> li.wp-block-sensei-lms-course-results__lesson {
+				border: 1px solid #d9d9d9;
+				margin-bottom: var(--wp--preset--spacing--20);
+				border-radius: var(--wp--custom--button--border--radius);
```

This isn't a button so it really shouldn't use this var. It is very commonly used though so I've considered adding a custom var for these boxes.
```suggestion
				border-radius: var(--wp--custom--box--border--radius);
```

---

b172d41

---

Thanks! I'll update the other usages.